docs/api-brownie.rst

```diff
@@ -127,7 +131,7 @@ The following classes and methods are used to convert arguments supplied to ``Co
         0xFF
         >>> bytes_to_hex("Hello")
           File "brownie/types/convert.py", line 149, in bytes_to_hex
-            raise ValueError("'{}' is not a valid hex string".format(value))
+            raise ValueError("'{value}' is not a valid hex string".format(value))
```

Comment:
`f"'{value}' is not a valid hex string"`  - good eye noticing this was outdated tho